scipy/sparse/linalg/_propack/dpropack.pyf

```diff
@@ -0,0 +1,136 @@
+        end subroutine dlansvd
+
+        subroutine dlansvd_irl(which,jobu,jobv,m,n,dim,p,neig,maxiter,aprod,u,ldu,sigma,bnd,v,ldv,tolin,work,lwork,iwork,liwork,doption,ioption,info,dparm,iparm) ! in :_dpropack:PROPACK/double/dlansvd_irl.F
+            use __user__routines, aprod=>daprod
+            character*1 intent(in) :: which
+            character*1 intent(in) :: jobu
+            character*1 intent(in) :: jobv
+            integer intent(in) :: m
+            integer intent(in) :: n
+            integer intent(in) :: dim
+            integer intent(in) :: p
+            integer intent(in) :: neig
+            integer intent(in) :: maxiter
+            external aprod
+            double precision intent(in,out),dimension(ldu,dim+1) :: u
+            integer optional,check(shape(u,0)==ldu),depend(u) :: ldu=shape(u,0)
+            double precision intent(out),dimension(neig) :: sigma
+            double precision intent(out),dimension(neig) :: bnd
+            double precision intent(in,out),dimension(ldv,dim) :: v
+            integer optional,check(shape(v,0)==ldv),depend(v) :: ldv=shape(v,0)
+            double precision intent(in) :: tolin
+            double precision intent(in),dimension(lwork) :: work
+            integer optional,check(len(work)>=lwork),depend(work) :: lwork=len(work)
+            integer intent(in),dimension(liwork) :: iwork
+            integer optional,check(len(iwork)>=liwork),depend(iwork) :: liwork=len(iwork)
+            double precision intent(in),dimension(4) :: doption
+            integer intent(in),dimension(2) :: ioption
+            integer intent(out) :: info
+            double precision intent(in),dimension(*) :: dparm
+            integer intent(in),dimension(*) :: iparm
+            integer :: nopx
+            integer :: nreorth
+            integer :: ndot
+            integer :: nreorthu
+            integer :: nreorthv
+            integer :: nitref
+            integer :: nrestart
+            integer :: nbsvd
+            real :: tmvopx
+            real :: tgetu0
+            real :: tupdmu
+            real :: tupdnu
+            real :: tintv
+            real :: tlanbpro
+            real :: treorth
+            real :: treorthu
+            real :: treorthv
+            real :: telru
+            real :: telrv
+            real :: tbsvd
+            real :: tnorm2
+            real :: tlansvd
+            integer :: nlandim
+            real :: tritzvec
+            real :: trestart
+            real :: tdot
+            integer :: nsing
+            common /timing/ nopx,nreorth,ndot,nreorthu,nreorthv,nitref,nrestart,nbsvd,tmvopx,tgetu0,tupdmu,tupdnu,tintv,tlanbpro,treorth,treorthu,treorthv,telru,telrv,tbsvd,tnorm2,tlansvd,nlandim,tritzvec,trestart,tdot,nsing
+        end subroutine dlansvd_irl
+    end interface
+end python module _dpropack
+
+! This file was auto-generated with f2py (version:2).
```

Comment:
Just to be sure that we want to have these files in (I don't know). Is this not generated like .c from Cython?

Reply:
@mckib2 ?

Reply:
Sorry - this comment slipped past my radar.  These pyf files are generated by numpy's `f2py` utility from the Fortran sources, but they are meant to be source controlled as there is extra contextual information that has been added.   So yes, they should be part of this PR.  The general workflow that I am familiar with goes like this:

    Run `f2py` on Fortran sources -> annotate, add information to the generated `pyf` files -> use annotated `pyf` files during the build